core/src/main/java/org/infinispan/configuration/cache/BackupConfigurationBuilder.java

```diff
@@ -148,6 +148,9 @@ public void validate() {
          throw new CacheConfigurationException("It is required to specify a 'failurePolicyClass' when using a " +
                                                 "custom backup failure policy!");
       }
+      if (useTwoPhaseCommit && strategy == BackupConfiguration.BackupStrategy.ASYNC) {
+         throw new CacheConfigurationException("'two-phase-commit' can be used only with SYNC backup strategy!");
```

Comment:
I was looking at the issue a bit more and found that data get replicated to remote site, however it seems1PC is used with 'ASYNC' + '2PC=true'.  (https://github.com/infinispan/infinispan/blob/master/core/src/main/java/org/infinispan/xsite/BackupSenderImpl.java#L160) - calculateBackupInfo(BackupFilter.KEEP_1PC_ONLY) does not return empty 'backups' list.

What about other 2 exceptions (failure policy, empty site)? Should I leave them as they are, or rewrite them using Log as well?